src/floki_selector_lexer.xrl

```diff
@@ -22,6 +22,7 @@ Rules.
 \({ODD}\)                            : {token, {pseudo_class_odd, TokenLine}}.
 \({EVEN}\)                           : {token, {pseudo_class_even, TokenLine}}.
 \({PSEUDO_PATT}\)                    : {token, {pseudo_class_pattern, TokenLine, remove_wrapper(TokenChars)}}.
+\({QUOTED}\)                         : {token, {pseudo_class_quoted, TokenLine, remove_wrapper(remove_wrapper(TokenChars))}}.
```

Comment:
I don't think so.

It seemed cleaner to me to disambiguate nested css expressions from literals/strings, so I made `:psuedo_class_quoted`, and also nicer-looking inside `selector_parser`.